src/blueapi/service/interface.py

```diff
@@ -63,8 +68,12 @@ def messaging_template() -> MessagingTemplate | None:
                 task_worker.data_events: event_topic,
             }
         )
-        template.connect()
-        return template
+        try:
+            template.connect()
+            return template
+        except Exception as ex:
+            logging.exception(msg="Failed to connect to message bus", exc_info=ex)
+            return None
```

Comment:
This except block changes the behaviour when a message bus is configured, but the message bus configuration is not valid.

Previously this was an error state that resulted in the failed startup of the service, which means on Kubernetes it would fall over, hopefully the message bus would recover, then this service could try and connect again. Now it will happily not create a connection in this case, which means on Kubernetes we will have created a declarative configuration but are not able to deliver that valid configuration.  I think this should be reverted and allowed to throw the exception here.